ttml/profile/dapt/Overview.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Dubbing and Audio description Profiles of TTML2 :: Namespaces</title>
+  </head>
+
+  <body>
+    <h1></h1>
+
+<p>This is a placeholder for a namespace document that permits dereferencing the Dubbing and Audio description Profiles of TTML2 namespace URIs specified at https://www.w3.org/TR/dapt/ :</p>
+
+<pre>
+http://www.w3.org/ns/ttml/profile/dapt#metadata
+http://www.w3.org/ns/ttml/profile/dapt/extension/
+</pre>
+
```

Comment:
I would prefer to see an element containing `id="metadata"` so that the first URI dereferences to a specific element. It's not essential though.